demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/backoff_algorithm.c

```diff
@@ -46,7 +46,7 @@ BackoffAlgorithmStatus_t BackoffAlgorithm_GetNextBackoff( BackoffAlgorithmContex
 
     /* If BACKOFF_ALGORITHM_RETRY_FOREVER is set to 0, try forever. */
```

Comment:
In the L475 sample, this looks like this, and I'm wondering if that should be the case here as well instead of my change in the code to match the existing comment.
```
/* If maxRetryAttempts state of the context is set to the maximum, retry forever. */
if( ( pRetryContext->maxRetryAttempts == BACKOFF_ALGORITHM_RETRY_FOREVER ) ||
    ( pRetryContext->attemptsDone < pRetryContext->maxRetryAttempts ) )
{